packages/back-end/src/services/organizations.ts

```diff
@@ -858,15 +858,15 @@ export async function importConfig(
               ...d,
             };
             delete updates.organization;
-
-            await updateDimension(k, organization.id, updates);
+            await updateDimension(context, existing, updates);
           } else {
             await createDimension({
```

Comment:
I'm confused why we're ultimately calling `createDimension` for config.yml managed dimensions - if an org uses the config.yml file, when we do a lookup we're looking for it within the config.yml file directly, so why are we making a `createDimension` call at all here?